### PR TITLE
Refactor the documentation for compatibility with the docs.scs.community

### PR DESCRIPTION
**Pull request by matofeder**
related to https://github.com/SovereignCloudStack/docs/issues/136

### CODE REVIEW

File: docs/backup_and_restore.md [hunk 1]
```diff
@@ -189,7 +189,7 @@ Note that the following backup steps mainly point to actions from an official [B
 In this guide, find the added value from additional explanations/hints and up-to-date commands.
 
 Harbor, by design, consists of multiple (micro)services that could store their data
-variously, based on the Harbor configuration. See the [Harbor persistence](harbor_persistence.md)
+variously, based on the Harbor configuration. See the [Harbor persistence](persistence)
```
Comment (michal-gubricky) [inline docs/backup_and_restore.md]:
```suggestion
variously, based on the Harbor configuration. See the [Harbor persistence](persistence.md)
```

**Review by michal-gubricky — CHANGES_REQUESTED** [review]

File: docs/migration.md [hunk 2]
```diff
@@ -23,7 +23,7 @@ are used for Harbor deployment, hence it is convenient to use them.
 ## Prerequisites
 
 As this scenario uses a backup and restore procedure for the migration process, this section
-refers to the prerequisites section in [Harbor - backup and restore](harbor_backup_restore.md#prerequisites)
+refers to the prerequisites section in [Harbor - backup and restore](backup_and_restore#prerequisites)
```
Comment (michal-gubricky) [inline docs/migration.md]:
```suggestion
refers to the prerequisites section in [Harbor - backup and restore](backup_and_restore.md#prerequisites)
```

File: docs/ha-deployment.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+
+```
+envs/public-ha/redis/redis-secret.bash # pwgen needs to be installed
+kubectl apply -k envs/public-ha/redis/
+kubectl apply -k envs/public-ha/postgres/
+```
+
+## Install Harbor
+
+- Take *ingress-nginx-controller* LoadBalancer IP address and create DNS record for Harbor.
```
Comment (chess-knight) [inline docs/ha-deployment.md:50]: Please, update `harbor-config.yaml` here and remove `registry.scs.dnation.cloud` url - we can instruct the user to change that in this step

File: docs/migration.md [hunk 4]
```diff
@@ -57,19 +57,19 @@ with a feature flag `--features=EnableAPIGroupVersions`. Note that this feature
 ### S3 bucket and EC2 credentials
 
 This guide assumes that the public cloud's object store with S3-compatible API is available as
-the storage backend for Velero. Refer to the [S3 bucket and EC2 credentials](harbor_backup_restore.md#s3-bucket-and-ec2-credentials)
+the storage backend for Velero. Refer to the [S3 bucket and EC2 credentials](backup_and_restore#s3-bucket-and-ec2-credentials)
```
Comment (michal-gubricky) [inline docs/migration.md]:
```suggestion
the storage backend for Velero. Refer to the [S3 bucket and EC2 credentials](backup_and_restore.md#s3-bucket-and-ec2-credentials)
```

File: docs/migration.md [hunk 5]
```diff
@@ -57,19 +57,19 @@ with a feature flag `--features=EnableAPIGroupVersions`. Note that this feature
...
 section in the backup and restore docs and create a bucket that will be later used for
 the migration.
 
 ### Velero client
 
-Install the Velero client on your local environment, refer to [Velero client](harbor_backup_restore.md#velero-client).
+Install the Velero client on your local environment, refer to [Velero client](backup_and_restore#velero-client).
```
Comment (michal-gubricky) [inline docs/migration.md]:
```suggestion
Install the Velero client on your local environment, refer to [Velero client](backup_and_restore.md#velero-client).
```

File: docs/migration.md [hunk 6]
```diff
@@ -57,19 +57,19 @@ with a feature flag `--features=EnableAPIGroupVersions`. Note that this feature
...
 
 ### Velero server
 
 Install Velero server components along with the appropriate plugins, into the both
 (Cluster_A and Cluster_B) clusters. Keep in mind that we will use Restic uploader in
-Velero, hence follows interactions on how to do that in the related docs section [Velero server](harbor_backup_restore.md#velero-server).
+Velero, hence follows interactions on how to do that in the related docs section [Velero server](backup_and_restore#velero-server).
```
Comment (michal-gubricky) [inline docs/migration.md]:
```suggestion
Velero, hence follows interactions on how to do that in the related docs section [Velero server](backup_and_restore.md#velero-server).
```

File: docs/migration.md [hunk 7]
```diff
@@ -101,7 +101,7 @@ $ kubectl -n velero --kubeconfig <path of Cluster_B kubeconfig> edit backupstora
 ## Migration
 
 Before we start to migrate Harbor instance from one environment to another go through the 
-[Backup and restore](harbor_backup_restore.md#backup-and-restore) section and read 
+[Backup and restore](backup_and_restore#backup-and-restore) section and read
```
Comment (michal-gubricky) [inline docs/migration.md]:
```suggestion
[Backup and restore](backup_and_restore.md#backup-and-restore) section and read 
```

**Review by michal-gubricky — CHANGES_REQUESTED** [review]

File: docs/ha-deployment.md [hunk 8]
```diff
@@ -0,0 +1,73 @@
...
+  ```
+  kubectl get svc -n ingress-nginx
+  NAME                                 TYPE           CLUSTER-IP      EXTERNAL-IP      PORT(S)                      AGE
+  ingress-nginx-controller             LoadBalancer   100.92.14.168   81.163.194.219   80:30799/TCP,443:32482/TCP   2m51s
+  ingress-nginx-controller-admission   ClusterIP      100.88.40.231   <none>           443/TCP                      2m51s
+  ```
+
+- Generate secrets and install Harbor:
```
Comment (chess-knight) [inline docs/ha-deployment.md:58]: Please, update `issuer.yaml` here and remove `roman.hros@dnation.cloud` email - we can instruct the user to change that in this step

File: docs/upgrade.md [hunk 9]
```diff
@@ -1,11 +1,11 @@
-# Harbor - upgrade
+# Upgrade
 
 This page aims at providing additional information for upgrading [Harbor](https://goharbor.io/)
 container registry, which operates in the Kubernetes environment and is deployed with Helm.
 It extends the official [Upgrading Harbor Deployed with Helm page](https://goharbor.io/docs/main/administration/upgrade/helm-upgrade/),
 where the upgrade process is well described. See the following upgrade hints:
 
-- Always [backup](harbor_backup_restore.md) your Harbor instance before upgrade
+- Always [backup](backup_and_restore) your Harbor instance before upgrade
```
Comment (michal-gubricky) [inline docs/upgrade.md]:
```suggestion
- Always [backup](backup_and_restore.md) your Harbor instance before upgrade
```

**Review by michal-gubricky — CHANGES_REQUESTED** [review]

File: docs/ha-deployment.md (hunk 3 above)
Comment (matofeder) [reply docs/ha-deployment.md:50]: fixed d0f77c382cb2216e5fc82a4939666d5810c2ec61

File: docs/ha-deployment.md (hunk 8 above)
Comment (matofeder) [reply docs/ha-deployment.md:58]: fixed d0f77c382cb2216e5fc82a4939666d5810c2ec61

**Review by michal-gubricky — APPROVED** [review]
lgtm

**Review by chess-knight — APPROVED** [review]
lgtm

**Pull request merged**